includes/class-wc-payments-account.php

```diff
@@ -18,7 +18,7 @@
  */
 class WC_Payments_Account {
 
-	const ACCOUNT_TRANSIENT              = 'wcpay_account_data';
+	const ACCOUNT_OPTION                 = 'wcpay_account_data';
```

Comment:
I'm wondering if we should leave `ACCOUNT_TRANSIET` defined for now (https://github.com/Automattic/woocommerce-payments/pull/1668/commits/b4e7fdc123e8d923ef39ac5e8931146db336ddfb). Removing it will brick developer sites until they update the dev tools plugin, it's a bit annoying so we could remove this later once everyone is upgraded?